save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityTagsComponent.kt

```diff
@@ -106,13 +111,18 @@ val vulnerabilityTagsComponent: FC<VulnerabilityTagsComponentProps> = FC { props
     div {
         className = ClassName("col pl-0")
         props.tags.forEach { tag ->
+            val tagEditable = props.userInfo?.name == props.vulnerability.name
```

Comment:
is a tag editable when userName is equal to vulnerability name?

Reply:
fixed